src/auth/users.controller.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Post,
+  ValidationPipe,
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { CreateUserDto } from './input/create.user.dto';
+import { User } from './entities/user.entity';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+
+@Controller('users')
+export class UsersController {
+  constructor(
+    private readonly authService: AuthService,
+    @InjectRepository(User)
+    private readonly userRepository: Repository<User>,
+  ) {}
+
+  @Post()
+  async create(@Body(new ValidationPipe()) createUserDto: CreateUserDto) {
```

Comment:
probably you can name this variable a different way in order not to confuse the code and devs who might read this code.

Reply:
Done